README.md

```diff
@@ -180,7 +180,18 @@ The cairo-vm-cli supports the following optional arguments:
 
 - `--allow_missing_builtins`: Disables the check that all builtins used by the program need to be included in the selected layout. Enabled by default when in proof_mode.
 
-- `run_from_cairo_pie`: Runs a Cairo PIE instead of a compiled json file. The name of the file will be the first argument received by the CLI (as if it were to run a normal compiled program). Can only be used if proof_mode is not enabled.
+- `--run_from_cairo_pie`: Runs a Cairo PIE instead of a compiled json file. The name of the file will be the first argument received by the CLI (as if it were to run a normal compiled program). Can only be used if proof_mode is not enabled.
+
+  ```bash
+  # First, compile a Cairo 0 program and run it to generate a Cairo PIE file
+  cairo-compile cairo_programs/fibonacci.cairo --output cairo_programs/fibonacci_compiled.json
+  target/release/cairo-vm-cli cairo_programs/fibonacci_compiled.json --layout all_cairo --cairo_pie_output fibonacci.pie
+  
+  # Then, run the program from the Cairo PIE file
+  target/release/cairo-vm-cli fibonacci.pie --run_from_cairo_pie --layout all_cairo --print_output
```

Comment:
Thanks for contributing, I would change the example program to one that uses the output builtin so that the CLI has a program output to print in console. For example, `print.cairo` uses the output builtin so we can change this to: 

```suggestion
  cairo-compile cairo_programs/print.cairo --output cairo_programs/print_compiled.json
  target/release/cairo-vm-cli cairo_programs/print_compiled.json --layout all_cairo --cairo_pie_output print.pie
  
  # Then, run the program from the Cairo PIE file
  target/release/cairo-vm-cli print.pie --run_from_cairo_pie --layout all_cairo --print_output
```